ops/docker-compose.yml

```diff
@@ -350,29 +353,6 @@ services:
       << : *integration_2_pk
       << : *integration_3_pk
 
-  gas_oracle:
```

Comment:
I've moved gas_oracle and monitor into side docker compose files as they put a dent on overall performance, but they're not needed in the integration tests as of now.